charts/gardener-metrics-exporter/charts/runtime/templates/deployment.yaml

```diff
@@ -66,10 +66,24 @@ spec:
         ports:
         - name: port
           containerPort: {{ .Values.global.server.port }}
+        {{- if .Values.global.resources }}
         resources:
-          requests:
-            cpu: 100m
-            memory: 128Mi
+          {{- toYaml .Values.global.resources | nindent 10 }}
+        {{- end }}
+        {{- if .Values.global.securityContext }}
+        securityContext:
+          {{- toYaml .Values.global.securityContext | nindent 10 }}
+        {{- end }}
```

Comment:
```suggestion
```

This part could also be removed, unless there is a use case for specifying a security context attribute.

Reply:
I would like to keep the possibility of setting securityContext attributes.
For example: We would like to set "readOnlyRootFilesystem: true" for our deployment.

Reply:
Thank you! That is a valid use case! Could you please add that to the `values.yaml`?

Reply:
Added securityContext.readOnlyRootFilesystem to true by default :)